js/page-specific/animal-services/adopted-pets.js

```diff
@@ -1,69 +1,68 @@
 (function ($, BcList) {
-  /**
+	/**
    * MODIFY these values based on your configuration
    */
-  var listClass = 'adoptable-pet-list';
-  var tabSelector = '.adoptable-pet-tabs';
-  var dataUrl = "//egov.baltimorecountymd.gov/LostAdoptPetService/GetAdoptablePets";
-  var templateUrl = "/sebin/l/l/generic-pet.template.js?version=20170728";
+	var listClass = 'adoptable-pet-list';
+	var tabSelector = '.adoptable-pet-tabs';
+	var dataUrl = '//bcgis.baltimorecountymd.gov/LostAdoptPetService/GetAdoptablePets';
+	var templateUrl = '/sebin/l/l/generic-pet.template.js?version=20170728';
 
-  /**
+	/**
    * !!!! DO NOT MODIFY BELOW THESE LINES !!!!
    */
 
-  //Initialize Bc List
-  var adoptedPets = new BcList({
-    containerClass: listClass, //Selector of class you want to append your results
-    source: dataUrl,
-    templatePath: templateUrl
-  });
+	// Initialize Bc List
+	var adoptedPets = new BcList({
+		containerClass: listClass, // Selector of class you want to append your results
+		source: dataUrl,
+		templatePath: templateUrl
+	});
 
-  //Utils
-  function getURLParameter(name) {
-    return decodeURI(
-      (RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
-    );
-  }
+	// Utils
+	function getURLParameter(name) {
+		return decodeURI(
+			(RegExp(name + '=' + '(.+?)(&|$)').exec(location.search) || [, null])[1]
+		);
+	}
 
-  function onReady() {
-    var type = getURLParameter('type') === "null" ? "all" : getURLParameter('type');
+	function onReady() {
+		var type = getURLParameter('type') === 'null' ? 'all' : getURLParameter('type');
 
-    if (type !== "all") {
-      var $adoptedPets = $(tabSelector + ' li');
-      $adoptedPets.find('a.active').removeClass('active');
-      $adoptedPets.find('a[data-type=' + type + ']').addClass('active');
-    }
+		if (type !== 'all') {
+			var $adoptedPets = $(tabSelector + ' li');
+			$adoptedPets.find('a.active').removeClass('active');
+			$adoptedPets.find('a[data-type=' + type + ']').addClass('active');
+		}
 
-    adoptedPets.Show(type, function () {
-      //If there is hash, scroll to that element
-      if (window.location.hash) {
-        document.getElementById(window.location.hash.replace("#", "")).scrollIntoView(true);
-      }
-    });
-  }
+		adoptedPets.Show(type, function () {
+			// If there is hash, scroll to that element
+			if (window.location.hash) {
+				document.getElementById(window.location.hash.replace('#', '')).scrollIntoView(true);
```

Comment:
Possible exception if `document.getElementById(window.location.hash.replace('#', ''))` is undefined.

Reply:
fixed